scripts/initialize/process_sample_freeze.py

```diff
@@ -0,0 +1,103 @@
+"""
+Gregory Way 2017
+PanCancer Classifier
+scripts/initialize/process_sample_freeze.py
+
+Takes in sample freeze data that was determined by TCGA PanCancer Atlas
+consortium along with raw RNAseq and mutation data. The script will process
+the datasets and subset each according to the frozen samples.
+
+Usage: Run once in command line
+
+        python scripts/initialize/process_sample_freeze.py
+
+Output:
+RNAseq and mutation data subset by sample freeze
+"""
+
+import os
+import numpy as np
+import pandas as pd
+
+# Input Files
+rna_file = os.path.join('data', 'raw', 'pancan_normalized_rnaseq.tsv')
+mut_file = os.path.join('data', 'raw', 'mc3.v0.2.8.PUBLIC.maf')
+sample_freeze_file = os.path.join('data', 'raw',
+                                  'sampleset_freeze_version4_modify.csv')
+
+# Output Files
+rna_out_file = os.path.join('data', 'pancan_rnaseq_freeze.tsv')
+mut_out_file = os.path.join('data', 'pancan_mutation_freeze.tsv')
+freeze_out_file = os.path.join('data', 'sample_freeze.tsv')
+
+# Load Data
+rnaseq_df = pd.read_table(rna_file, index_col=0)
+mutation_df = pd.read_table(mut_file)
+sample_freeze_df = pd.read_csv(sample_freeze_file)
+
+# Process RNAseq file
+rnaseq_df.index = rnaseq_df.index.map(lambda x: x.split('|')[0])
+rnaseq_df.columns = rnaseq_df.columns.str.slice(start=0, stop=15)
+rnaseq_df = rnaseq_df.drop('?').fillna(0).sort_index(axis=1)
+rnaseq_df.drop('SLC35E2', axis=0, inplace=True)
```

Comment:
its a gene that was listed twice in the rnaseq data - must have been a mistake in data processing since in many TCGA datasets I've worked with, it causes this problem. I added a comment in the next commit, thanks!